core/src/main/scala/org/broadinstitute/dsde/rawls/fastpass/FastPassMonitor.scala

```diff
@@ -18,27 +17,20 @@ import org.broadinstitute.dsde.workbench.model.WorkbenchEmail
 object FastPassMonitor {
   sealed trait FastPassMonitorMessage
   case object DeleteExpiredGrants extends FastPassMonitorMessage
-  def props(dataSource: SlickDataSource, googleIamDAO: GoogleIamDAO, googleStorageDao: GoogleStorageDAO)(implicit
-    openTelemetry: OpenTelemetryMetrics[IO]
-  ): Props = Props(
+  def props(dataSource: SlickDataSource, googleIamDAO: GoogleIamDAO, googleStorageDao: GoogleStorageDAO): Props = Props(
     new FastPassMonitor(dataSource, googleIamDAO, googleStorageDao)
   )
 }
 
 class FastPassMonitor private (dataSource: SlickDataSource,
                                googleIamDAO: GoogleIamDAO,
                                googleStorageDao: GoogleStorageDAO
-)(implicit openTelemetry: OpenTelemetryMetrics[IO])
-    extends Actor
+) extends Actor
     with LazyLogging {
   import context.dispatcher
 
-  private val openTelemetryTags: Map[String, String] = Map("service" -> "FastPassMonitor")
   override def receive: Receive = { case DeleteExpiredGrants =>
-    for {
-      _ <- openTelemetry.incrementCounter("fastpass-monitor-sweeper-start", tags = openTelemetryTags).unsafeToFuture()
```

Comment:
note that I removed this metric because it has no value - this function is called on a schedule every 10s